gtfs/CHANGES.md

```diff
@@ -50,25 +50,116 @@ Every new feature adds complexity to the creation and reading of feeds. Therefor
 
 ### Revision History
 
-#### January 17, 2019
+#### May 28, 2020
 
-* Editorial and formatting changes for clarity.  See [pull request](https://github.com/google/transit/pull/120).
+* Updated major GTFS changes under "Revision History" in `CHANGES.md`. Last revised in January 2019 with some historical entries missing.
+
+#### May 13, 2020
+
+* Added `continuous_pickup` and `continuous_drop_off` to `routes.txt` and `stop_times.txt`.  See [discussion](https://github.com/google/transit/pull/208).
+* Changed `shape_id` from "Optional" to "Conditionally required". See [discussion](https://github.com/google/transit/pull/208).
+
+#### March 24, 2020
+
+* Defined text-to-speech field and added `tts_stop_name` to `stops.txt`. See [discussion](https://github.com/google/transit/pull/49).
+
+#### March 20, 2020
+
+* Added recommendation for handling TripUpdate predictions for multiple trips in the same block. See [discussion](https://github.com/google/transit/pull/206).
```

Comment:
This file has historically been used only for static GTFS revision history. 

However, unless I'm missing something, it looks like the Google website that has the GTFS-RT revision history:
https://developers.google.com/transit/gtfs-realtime/guides/revision-history

...never made it into the GitHub repo.  There is an equivalent CHANGES.md for GTFS-RT:
https://github.com/google/transit/blob/master/gtfs-realtime/CHANGES.md

But it's missing the revision history.

So we need to decide if we keep all GTFS and GTFS-RT revisions in the same document, or if we split them across two files.

In any case, the Google website revision history should be migrated into this GitHub repo somewhere.

Reply:
Good catch. I meant to include only the updated revision history for static GTFS only. 

I noticed as well that there was no GTFS-RT revision history at https://github.com/google/transit/blob/master/gtfs-realtime/CHANGES.md

I would be for adding it there.